src/model/entities/segment.ts

```diff
@@ -58,6 +61,18 @@ export class Segment {
     this.isSegmentOnAir = false
   }
 
+  public setExecutedAtEpochTime(executedAtEpochTime: number): void {
+    this.executedAtEpochTime ??= executedAtEpochTime
+  }
+
+  public clearExecutedAtEpochTime(): void {
+    this.executedAtEpochTime = undefined
+  }
+
+  public getExecutedAtEpochTime(): number | undefined {
+    return this.executedAtEpochTime
+  }
```

Comment:
No need to expose these methods if you just let the Segment handle its own executedAt state in putOnAir and takeOfAir

Reply:
These no longer exist.